src/test/java/org/folio/des/scheduling/acquisition/AcqBaseExportTaskTriggerTest.java

```diff
@@ -369,7 +369,6 @@ public void weeklyTestScheduleOnceWithMultipleChosenDaysAndFrequencyWhenDayFromN
     ZonedDateTime actDateTime = getActualTime(actDate);
     assertEquals(scheduledDateTime.getHour(), actDateTime.getHour());
     assertEquals(dayBefore, actDateTime.getDayOfWeek());
-    assertNotEquals(scheduledDateTime.get(IsoFields.WEEK_OF_WEEK_BASED_YEAR), actDateTime.get(IsoFields.WEEK_OF_WEEK_BASED_YEAR));
```

Comment:
This deletion is expected, test has floating issue to fail when current day is Monday, deleted this check that has this floating issue